src/openvic-simulation/map/Map.hpp

```diff
@@ -139,7 +139,8 @@ namespace OpenVic {
 		bool load_province_definitions(std::vector<ovdl::csv::LineObject> const& lines);
 		/* Must be loaded after adjacencies so we know what provinces are coastal, and so can have a port */
 		bool load_province_positions(BuildingTypeManager const& building_type_manager, ast::NodeCPtr root);
-		bool load_region_file(ast::NodeCPtr root);
+		bool load_region_colours(ast::NodeCPtr root, std::vector<colour_t>& colours);
```

Comment:
Since this function doesn't use any of Map's member variables/functions, it could be made `static` and then called in `Dataloader.cpp` via `Map::load_region_colours`.

Reply:
oops, I hadn't even thought about it, thanks for the feedback :)